### ISSUE TITLE
Continuosly freeze

### ISSUE THREAD

**Issue by Vincio1991** [qgis/QGIS#44899]
### What is the bug or the crash?

After I upgraded my GPU (I had GTX 1060 and now I have RTX 3060) QGIS continuosly freeze, even for simple tasks like opening properties tab (sometimes it requires 15 sec to open), sometimes also opening every menu... I tried LT and QT version, both gave me same issues...
I tried with another PC and the problem is not linked to the project..
How can I solve?

### Steps to reproduce the issue

right click on layer, open properties tab

### Versions

Both LT 3.16 and QT 3.20.2

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

**Comment by gioman** [comment]
@Vincio1991 on what OS? have you really tried a new QGIS profile? 

>  I tried with another PC and the problem is not linked to the project..

does it means that on a different pc works ok?

**Comment by Vincio1991** [comment]
> @Vincio1991 on what OS? have you really tried a new QGIS profile?
> 
> > I tried with another PC and the problem is not linked to the project..
> 
> does it means that on a different pc works ok?

Yes, it works properly on my 2nd PC. On my main PC I uninstalled Qgis, then I removed all files and registry to perform a clean install of the program, but didn't work

**Comment by Vincio1991** [comment]
I'm using Windows 10 btw

**Comment by Vincio1991** [comment]
My main PC has this spec:
Windows 10 (last update)
i5 10600kf @4700 MHz
RTX 3060 (no OC, latest driver)
32GB RAM @3000 MHz

**Comment by roya0045** [comment]
Have you enabled or disabled opencl?

**Comment by Vincio1991** [comment]
> Have you enabled or disabled opencl?

tried  both, same issues

**Comment by gioman** [comment]
> but didn't work

@Vincio1991 settings > user profiles > new profile

**Comment by Vincio1991** [comment]
> > but didn't work
> 
> @Vincio1991 settings > user profiles > new profile

already done, didn't solve :/

**Comment by gioman** [comment]
> already done, didn't solve :/ 

@Vincio1991 can you test with the old graphic card?

**Comment by Vincio1991** [comment]
Nope, I don't have it anymore... :/

**Comment by gioman** [comment]
> Nope, I don't have it anymore... :/ 

@Vincio1991 do you have a motherboard with on board graphic card?

**Comment by Vincio1991** [comment]
> > Nope, I don't have it anymore... :/
> 
> @Vincio1991 do you have a motherboard with on board graphic card?

it depends on CPU, and I don't have one with integrated GPU. It's an i5 F version
